tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/OffloadIndexBlock.java

```diff
@@ -73,7 +71,7 @@
     /**
      * An input stream which knows the size of the stream upfront.
      */
-    public static class IndexInputStream extends FilterInputStream {
+    class IndexInputStream extends FilterInputStream {
```

Comment:
this should still be static

Reply:
Good catch. I will correct.

Reply:
The reason I removed both the "public" and "static" modifiers for this class, was due to the fact that check style flagged them both as redundant.  

According to the java 8 spec, https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.5;

"Interfaces may contain member type declarations (§8.5).

A member type declaration in an interface is implicitly public and static."